shared/Microsoft.Extensions.WebEncoders.Sources/WebEncoders.cs

```diff
@@ -3,8 +3,15 @@
 
 using System;
 using System.Diagnostics;
-using System.Globalization;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
 using Microsoft.Extensions.WebEncoders.Sources;
+using System.Buffers;
+using System.Buffers.Text;
+
+#if !NET461
+using System.Numerics;
```

Comment:
`Vector<T>` isn't part of net461, so I've `#if`ed it out. Is there a way to bring `Vector<T>` to net461 in this project (linked file) or shouldn't we don't care about this :wink: